File: 0001_0100/0062/0062.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    s = f.read().strip()
    q.write("BLACK" if s[0] in "ACEG" and s[1] in "1357" or s[0] in "BDFH" and s[1] in "2468" else "WHITE")
        
    
           
        

File: 0001_0100/0062/0062.go
package main

import (
	"fmt"
	"os"
)

func main() {
	f, _ := os.Open("input.txt")
	q, _ := os.Create("output.txt")
	var s string
	fmt.Fscan(f, &s)
	r := "BLACK"
	if (s[0]-'A'+s[1]-'0')%2 == 0 {
		r = "WHITE"
	}
	fmt.Fprint(q, r)
}
